packages/allure-cucumberjs/src/BufferEncodingUtility.ts

```diff
@@ -0,0 +1,8 @@
+export const getBufferEncoding = (encoding: any): BufferEncoding => {
```

Comment:
`encoding` should have `string` type

Reply:
And I think, better to rename the function to `normalizeBufferEncoding` because we use it as adapter or normalizer, not getter

Reply:
I believe BufferEncoding already is type string.

```
  writeAttachment(name: string, content: Buffer | string, encoding?: BufferEncoding): void {
    const enc = "base64";
    encoding = enc;
    ```
    
    The following works fine.

Reply:
unnecessary file